packages/react-table/src/components/TableComposable/Td.tsx

```diff
@@ -227,6 +250,15 @@ const TdBase: React.FunctionComponent<TdProps> = ({
       {mergedChildren || children}
     </MergedComponent>
   );
+
+  const canDefault = tooltip === '' ? typeof children === 'string' : true;
```

Comment:
Is this variable name accurate? I'm not sure if it's descriptive for its purpose or if I'm not interpreting it correctly.

Reply:
I matched what was present in `Th`, I'll look at it some more to see if there's a better name.